backend/apps/owasp/Makefile

```diff
@@ -33,6 +33,10 @@ owasp-update-project-health-metrics:
 	@echo "Updating OWASP project health metrics"
 	@CMD="python manage.py owasp_update_project_health_metrics" $(MAKE) exec-backend-command
 
+owasp-sync-official-project-levels:
```

Comment:
This is not right coupling decision. I suggested to implement the official project levels during the project data sync. The health metrics is the next step after data sync is done.